ua-parser.js

```diff
@@ -47,8 +48,19 @@ const parseUA = (userAgent, browsers) => {
     data.browser.name += ' Android';
   }
 
+  data.fullVersion = ua.browser.version;
   // https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#safari-for-ios-versioning
-  data.fullVersion = data.browser.id === 'safari_ios' ? ua.os.version : ua.browser.version;
+  if (data.browser.id === 'safari_ios') {
+    data.fullVersion = ua.os.version;
+  }
+
+  if (ua.browser.name === 'Android Browser') {
+    if (compareVersions.compare(ua.os.version, '5.0', '<')) {
```

Comment:
I'm pretty sure Chrome for Android 37 would have been available on Android 4.4.3, so is this the wrong check? How about instead checking if the engine is Chromium, with an assert that the version should be 37 or later?

Reply:
The documentation that describes WebView indicates that Android 4.4.3's browser was based upon Chrome 33.  At the top, it has a statement that indicates that WebView 37 wasn't available before Android 5:

> In Android 5.0 (Lollipop), the WebView has moved to an APK so it can be updated seperately to the Android platform.